packages/cursorless-engine/src/processTargets/targets/PositionTarget.ts

```diff
@@ -41,11 +48,7 @@ export default class PositionTarget extends BaseTarget {
   getRemovalRange = () => removalUnsupportedForPosition(this.position);
 
   getEditNewActionType(): EditNewActionType {
-    if (this.insertionDelimiter === "\n" && this.position === "after") {
-      return "insertLineAfter";
-    }
-
-    return "edit";
+    return this.editNewActionType;
```

Comment:
Any reason why you moved it from here to the constructor?

Reply:
For some reason I thought we no longer had access to the `thatTarget` at this point.  But in retrospect obviously we do 🤷‍♂️.  Nice catch